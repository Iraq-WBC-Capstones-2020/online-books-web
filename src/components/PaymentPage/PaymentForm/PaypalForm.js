import React from 'react';
import { MDBContainer, MDBRow, MDBCol, MDBInput, MDBBtn } from 'mdbreact';
import { useTranslation } from 'react-i18next';

export default function PaypalForm() {
  const { t } = useTranslation();
  return (
    <MDBContainer>
      <MDBRow>
        <MDBCol md="12">
          <form>
            <div className="grey-text">
              <MDBInput
                label={t('paymentPage.email')}
                icon="envelope"
                containerClass="d-flex flex-row-reverse"
                className="ml-0 mr-1"
                labelClass="ml-4.5"
                group
                type="email"
                validate
                error="wrong"
                success="right"
              />
              <MDBInput
                label={t('paymentPage.password')}
                icon="lock"
                containerClass="d-flex flex-row-reverse"
                className="ml-0 mr-1"
                labelClass="ml-4.5"
                group
                type="password"
                validate
              />
            </div>
            <div className="text-center mb-3">
              <MDBBtn
                type="button"
                color="blue"
                className="rounded-pill text-white z-depth-1a w-48"
              >
                {t('paymentPage.purchase')}
              </MDBBtn>
            </div>
          </form>
        </MDBCol>
      </MDBRow>
    </MDBContainer>
  );
}
