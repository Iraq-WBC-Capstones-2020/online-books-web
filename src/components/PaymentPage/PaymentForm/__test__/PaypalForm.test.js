import React from 'react';
import renderer from 'react-test-renderer';
import PaypalForm from './../PaypalForm';

test('PaypalForm snapshot', () => {
  const tree = renderer.create(<PaypalForm />).toJSON();
  expect(tree).toMatchSnapshot();
});
