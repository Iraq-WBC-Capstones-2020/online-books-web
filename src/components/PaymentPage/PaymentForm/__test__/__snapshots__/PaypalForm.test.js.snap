// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaypalForm snapshot 1`] = `
<div
  className="container"
  data-test="container"
>
  <div
    className="row"
    data-test="row"
  >
    <div
      className="col-md-12"
      data-test="col"
    >
      <form>
        <div
          className="grey-text"
        >
          <div
            className="md-form form-group d-flex flex-row-reverse"
          >
            <i
              className="fa fa-envelope prefix"
              data-test="fa"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
            />
            <input
              aria-disabled={false}
              className="form-control validate ml-0 mr-1"
              data-test="input"
              disabled={false}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onInput={[Function]}
              type="email"
              value=""
            />
            <label
              aria-labelledby=""
              className="ml-4.5"
              data-error="wrong"
              data-success="right"
              id=""
              onClick={[Function]}
            >
              paymentPage.email
            </label>
          </div>
          <div
            className="md-form form-group d-flex flex-row-reverse"
          >
            <i
              className="fa fa-lock prefix"
              data-test="fa"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
            />
            <input
              aria-disabled={false}
              className="form-control validate ml-0 mr-1"
              data-test="input"
              disabled={false}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onInput={[Function]}
              type="password"
              value=""
            />
            <label
              aria-labelledby=""
              className="ml-4.5"
              data-error=""
              data-success=""
              id=""
              onClick={[Function]}
            >
              paymentPage.password
            </label>
          </div>
        </div>
        <div
          className="text-center mb-3"
        >
          <button
            className="btn-blue btn Ripple-parent rounded-pill text-white z-depth-1a w-48"
            data-test="button"
            onMouseUp={[Function]}
            onTouchStart={[Function]}
            type="button"
          >
            paymentPage.purchase
            <div
              className="Ripple "
              data-test="waves"
              style={
                Object {
                  "height": "0px",
                  "left": "0px",
                  "top": "0px",
                  "width": "0px",
                }
              }
            />
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
